### DISCUSSION TITLE
CF Client running from Python crashes

### DISCUSSION THREAD

**Discussion by Itai51** [bitcraze/discussions#1361] [Q&A]
Hello Bitcraze community!

I am using a Windows 11 computer, and for a while I've been attempting to run the CF Client through the VM. However, it lags too much to be useful, even after I follow the VM optimization instructions.

I recently attempted running the CF Client natively from Python (3.12). The client appears to open with no problem. It even seems not to lag, however I cannot be sure, because as soon as I try to connect my Crazyflie, the GUI crashes. 

The only error I can seem to identify thus far is "too many packets lost", but the solutions I've seen so far on the various forums for that error haven't worked.

I am certain there is a quick fix (hoping!), and I would appreciate any troubleshooting help.

Thank you!

**Comment by knmcguire** [comment]
Hi! Could you share the content of the terminal of where you started the cfclient from?

I work daily on windows 11 and the latest cfclient on python 3.10, and so far it has been working ok for me, but there are instabilities. Perhaps some of these open tickets can describe the issue you are having:

- https://github.com/bitcraze/crazyflie-clients-python/issues/656
- https://github.com/bitcraze/crazyflie-clients-python/issues/690
